lib/apiservers/engine/backends/system.go

```diff
@@ -55,8 +55,10 @@ type System struct {
 }
 
 const (
-	systemStatusMhz    = " VCH mhz limit"
+	systemStatusMhz    = " VCH CPU limit"
 	systemStatusMemory = " VCH memory limit"
+	systemStatusCPUUsageMhz    = " VCH CPU usage"
```

Comment:
have you run go fmt? Please do.
in gogland you can change settings to run goimports on save.